images/kubekins-e2e/variants.yaml

```diff
@@ -16,8 +16,8 @@ variants:
     CONFIG: master
```

Comment:
please create a test-infra variant, do not corrupt the kubernetes/kubernetes variant

Reply:
Done.